pkg/scripts/bigchaindb-monit-config

```diff
@@ -110,6 +110,7 @@ case \$1 in
   start_tendermint)
 
     pushd \$4
+
       nohup tendermint node --consensus.create_empty_blocks=false >> \$3/tendermint.out.log 2>> \$3/tendermint.err.log &
```

Comment:
I'm not sure we need the `--consensus.create_empty_blocks=false` in the command to start Tendermint, because that's ensured by the settings we tell them to put in `config.toml`, i.e. see http://docs.bigchaindb.com/projects/server/en/master/simple-deployment-template/network-setup.html#member-connect-to-the-other-members